src/server.ts

```diff
@@ -21,7 +38,12 @@ export function configureApp(): express.Application {
   app.use('/api/v1', router);
   app.use('/api/v1/auth', authRoute);
   app.get('/', (req, res) => {
-    res.status(200).send('welcome to E-commerce API');
+    res.send(
+      '<a href="/api/v1/auth/google">Login to our E-commerce with Google</a>',
+    );
+  });
+  app.get('/home', isLoggedIn, (req, res) => {
+    res.send('Welcome to google auth');
```

Comment:
This should go in the passport.routes... Let's just initialize initialize passport and passport.session in this file